### PR TITLE
Write tests for edit, complete, and clear functionalities

### PR DESCRIPTION
**Pull request by c00p75**
Created test file to test the following functionalitites:
- A function for editing the task description.
- A function for updating an item's 'completed' status.
- The "Clear all completed" function.

### CODE REVIEW

**Comment by julie-ify** [comment]
Hi @c00p75,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### As this is a collaborative project kindly ensure your coding partner commits in this project

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/Tests/addedFunctionality.test.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
+/* @jest-environment jsdom*/
+/* eslint-disable */
```
Comment (emmiiorji) [inline src/Tests/addedFunctionality.test.js:2]: Please, remember also not to disable linter checks in-line. Linters represent coding style guidelines that you share with your team and altering the configurations or making efforts to by-pass them might lead to divergent coding styles in a codebase, thereby messing it up.

Discipline yourself now and you'll be glad you did.

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @c00p75  and @ohyajk, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Invalid2

You have submitted a project where you attempted to by-pass linter checks by creating a [.eslintrc.js](https://github.com/c00p75/To-Do-List/blob/unit-tests/.eslintrc.js) file. The is highly prohibited. Please, delete this file and correct all linter errors.


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by codecaiine — APPROVED** [review]
Hi @c00p75 @emmiiorji , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good testing implementation :heavy_check_mark: 
- No linters errors :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**